### DISCUSSION TITLE
public link

### DISCUSSION THREAD

**Discussion by MarinusPretorius** [iptv-org/community#16] [Q&A] [answered]
Is there not someone that expose this via a public link that I can just import in IPTV editor and also update the files on a daily basis.  

I can generate the site myself but its then a manual process every couple of days to copy the file to a sub domain folder.

IPTV Editor, best epg, epgshare, bevy all are very unstable in providing the correct epg.

**Comment by freearhey** [comment, marked as answer]
https://github.com/iptv-org/awesome-iptv?tab=readme-ov-file#epg-sources
